website/docs/docs/dbt-cloud/dbt-cloud-apis/cloud-apis.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "dbt Cloud APIs"
+id: "cloud-apis"
```

Comment:
can you just leave this `cloud-api`? This generates the URL for the page, and it would be better not to make a breaking change to this URL given that we link to it in lots of other places! We could add a redirect if we really needed to... but.... might be easier just to keep the old URL!